worker/Makefile

```diff
@@ -190,7 +190,9 @@ clean-pip:
 clean-subprojects: meson-ninja
 	$(MESON) subprojects purge --include-cache --confirm
 
-clean-all: clean-subprojects
+# Clean subprojects (ignore if it fails) and delete out dir.
+clean-all:
+	-$(MESON) subprojects purge --include-cache --confirm
```

Comment:
Yes, this minus before the command tells Make to ignore the error and keep working.